Features/Language.feature

```diff
@@ -0,0 +1,23 @@
+Feature: As a customer I can add edit and delete language 
+
+
+Scenario: Add new Language
+
+Given I navigate to Language
```

Comment:
please extend coverage for skill and language tests 


• Negative testing with valid input
• Negative testing with invalid input
• Destructive testing

- E.g of above scenarios
- Happy path tests check basic functionality and the acceptance criteria of the API. We later extend positive tests to include optional parameters and extra functionality.
- 
- The next group of tests is negative testing where we expect the application to gracefully handle problem scenarios with both valid user input (for example, trying to add an existing username) and invalid user input (trying to add a username which is null).
- 
- Destructive testing is a deeper form of negative testing where we intentionally attempt to break the API to check its robustness (for example, sending a huge payload body in an attempt to overflow the system).
- 
- If some scenarios does not make sense to add you can you should know why it does not make  sense to add.

Reply:
As suggested, I have covered both negative testing with valid & invalid input (please refer below)
https://github.com/Godzeye99/Mars_QA_repo/blob/onboarding_task/Features/Language.feature#L33
https://github.com/Godzeye99/Mars_QA_repo/blob/onboarding_task/Features/Skill.feature#L32

But for destructive testing, I need some helping in finding the correct scenario